src/bun.js/api/server.zig

```diff
@@ -4165,7 +4165,8 @@ pub const ServerWebSocket = struct {
         if (this.isClosed() or vm.isShuttingDown())
             return;
 
-        if (handler.onDrain != .zero) {
+        // we should not call drain if we dont have this_value
+        if (handler.onDrain != .zero and !this.this_value.isEmptyOrUndefinedOrNull()) {
```

Comment:
Seems kind of sketchy. Maybe we should call `getThisValue()` instead?

Reply:
Actually Im debugging to see where is the issue but `drain` event is called passing a null value, probably we are unprotecting to soon?

Reply:
Yeah `getThisValue` was not called before because I did not add a open callback, so getThisValue fix this issue